test/index_tests.js

```diff
@@ -224,10 +224,20 @@ it('WHEN package json includes baseUrlPath THEN spec description is updated', ()
     assert.equal(spec.info.description.indexOf(', base url :') > 0, true);
   }, 1001);
 
+  app.set('port', port);
```

Comment:
You don't need this code. Add `done()` to the next line after `assert.equal(spec.info.description.indexOf(', base url :') > 0, true);`. Same below.